ui/components/src/GaugeChart/GaugeChart.tsx

```diff
@@ -152,7 +152,10 @@ export function GaugeChart(props: GaugeChartProps) {
             color: 'inherit', // allows value color to match active threshold color
             fontSize: valueSizeClamp,
```

Comment:
I think this was an accident that got pulled in with a bad rebase/merge. Fix this after the holidays.